custom_components/ha-glocaltokens/manifest.json

```diff
@@ -1,10 +1,10 @@
 {
-  "domain": "ha-glocaltokens",
-  "name": "HA-Glocaltokens",
+  "domain": "glocaltokens",
+  "name": "Google local authentication token fetching (Glocaltokens)",
   "documentation": "https://github.com/leikoilja/ha-glocaltokens",
   "issue_tracker": "https://github.com/leikoilja/ha-glocaltokens/issues",
   "dependencies": [],
   "config_flow": true,
   "codeowners": ["@leikoilja"],
-  "requirements": ["glocaltokens==0.1.4"]
```

Comment:
The manifest must have the package version for us to be able to merge it in HACS according to their documentation.

Reply:
I dont know why i have removed it